aiida/orm/entities.py

```diff
@@ -170,12 +171,15 @@ def objects(cls: EntityType) -> Collection[EntityType]:  # pylint: disable=no-se
 
     @classmethod
     def get(cls, **kwargs):
+        """Get an entity of the collection matching the given filters."""
```

Comment:
Perhaps specify that is deprecated in the docstring

Reply:
(and similar for `id`)

Reply:
Done